docs/content/learn/tutorials/control/basic-profile.md

```diff
@@ -0,0 +1,1282 @@
+      props:
+        - name: label
+          value: 1.1.2
+      parts:
+        - id: s1.1.2_stm
+          name: statement
+          prose:
+            Conflicting duties and areas of responsibility should be segregated to
+            reduce opportunities for unauthorized or unintentional modification or misuse
+            of the organization’s assets.
+        - id: s1.1.2_gdn
+          name: guidance
+          parts:
+            - id: s1.1.2_gdn.1
+              name: item
+              prose:
+                Care should be taken that no single person can access, modify or use
+                assets without authorization or detection. The initiation of an event should
+                be separated from its authorization. The possibility of collusion should
+                be considered in designing the controls.
+            - id: s1.1.2_gdn.2
+              name: item
+              prose:
+                Small organizations may find segregation of duties difficult to achieve,
+                but the principle should be applied as far as is possible and practicable.
+                Whenever it is difficult to segregate, other controls such as monitoring
+                of activities, audit trails and management supervision should be considered.
+        - id: s1.1.2_inf
+          name: information
+          prose:
+            Segregation of duties is a method for reducing the risk of accidental
+            or deliberate misuse of an organization’s assets.
+    - id: s2.1.2
+      title: Access to networks and network services
+      props:
+        - name: label
+          value: 2.1.2
+      parts:
+        - id: s2.1.2_stm
+          name: statement
+          prose:
+            Users should only be provided with access to the network and network
+            services that they have been specifically authorized to use.
+        - id: s2.1.2_gdn
+          name: guidance
+          parts:
+            - id: s2.1.2_gdn.1
+              name: item
+              prose: |-
+                A policy should be formulated concerning the use of networks and network services. This policy should cover:
+
+                1. the networks and network services which are allowed to be accessed;
+                2. authorization procedures for determining who is allowed to access which networks and networked services;
+                3. management controls and procedures to protect access to network connections and network services;
+                4. the means used to access networks and network services (e.g. use of VPN or wireless network);
+                5. user authentication requirements for accessing various network services;
+                6. monitoring of the use of network service
+            - id: s2.1.2_gdn.2
+              name: item
+              prose:
+                The policy on the use of network services should be consistent with
+                the organization’s access control policy
```

Comment:
Updated catalog:

```suggestion
---
catalog:
  uuid: b1296da4-4ada-40f1-bbeb-20e3429bef41
  metadata:
    title: Sample Security Profile *for Demonstration* and Testing
    last-modified: 2023-05-13T22:08:16.76575041Z
    version: "1.0"
    oscal-version: 1.0.4
  controls:
  - id: s1.1.1
    title: Information security roles and responsibilities
    params:
    - id: s1.1.1-prm1
      label: a choice from a selection
      select:
        how-many: one-or-more
        choice:
        - "initiating a device lock after {{ insert: param, s1.1.1-prm_2 }} of inactivity"
        - requiring the user to initiate a device lock before leaving the system unattended
    - id: s1.1.1-prm_2
      label: a duration (maximum 30 minutes)
    props:
    - name: label
      value: 1.1.1
    parts:
    - id: s1.1.1_stm
      name: statement
      prose: |-
        All information security responsibilities should be defined and allocated.

        A value has been assigned to {{ insert: param, s1.1.1-prm1 }}.

        A cross link has been established with a choppy syntax: [(choppy)](#s1.2).
    - id: s1.1.1_gdn
      name: guidance
      parts:
      - id: s1.1.1_gdn.1
        name: item
        prose: "Allocation of information security responsibilities should be done in accordance with the information security policies. Responsibilities for the protection of individual assets and for carrying out specific information security processes should be identified. Responsibilities for information security risk management activities and in particular for acceptance of residual risks should be defined. These responsibilities should be supplemented, where necessary, with more detailed guidance for specific sites and information processing facilities. Local responsibilities for the protection of assets and for carrying out specific security processes should be defined."
      - id: s1.1.1_gdn.2
        name: item
        prose: Individuals with allocated information security responsibilities may delegate security tasks to others. Nevertheless they remain accountable and should determine that any delegated tasks have been correctly performed.
      - id: s1.1.1_gdn.3
        name: item
        prose: |-
          Areas for which individuals are responsible should be stated. In particular the following should take place:

          1. the assets and information security processes should be identified and defined;
          2. the entity responsible for each asset or information security process should be assigned and the details of this responsibility should be documented;
          3. authorization levels should be defined and documented;
          4. to be able to fulfil responsibilities in the information security area the appointed individuals should be competent in the area and be given opportunities to keep up to date with developments;
          5. coordination and oversight of information security aspects of supplier relationships should be identified and documented.
      - id: s1.1.1_gdn.4
        name: item
        prose: "Users of devices running Gnome can adjust the inactivity timeout using the following link: https://help.gnome.org/admin/system-admin-guide/stable/desktop-lockscreen.html.en"
    - id: s1.1.1_inf
      name: information
      props:
      - name: label
        value: Other information
      prose: |-
        Many organizations appoint an information security manager to take overall responsibility for the development and implementation of information security and to support the identification of controls.

        However, responsibility for resourcing and implementing the controls will often remain with individual managers. One common practice is to appoint an owner for each asset who then becomes responsible for its day-to-day protection.
  - id: s1.1.2
    title: Segregation of duties
    props:
    - name: label
      value: 1.1.2
    parts:
    - id: s1.1.2_stm
      name: statement
      prose: Conflicting duties and areas of responsibility should be segregated to reduce opportunities for unauthorized or unintentional modification or misuse of the organization’s assets.
    - id: s1.1.2_gdn
      name: guidance
      parts:
      - id: s1.1.2_gdn.1
        name: item
        prose: "Care should be taken that no single person can access, modify or use assets without authorization or detection. The initiation of an event should be separated from its authorization. The possibility of collusion should be considered in designing the controls."
      - id: s1.1.2_gdn.2
        name: item
        prose: "Small organizations may find segregation of duties difficult to achieve, but the principle should be applied as far as is possible and practicable. Whenever it is difficult to segregate, other controls such as monitoring of activities, audit trails and management supervision should be considered."
    - id: s1.1.2_inf
      name: information
      prose: Segregation of duties is a method for reducing the risk of accidental or deliberate misuse of an organization’s assets.
  - id: s2.1.1
    title: Access control policy
    props:
    - name: label
      value: 2.1.1
    parts:
    - id: s2.1.1_stm
      name: statement
      prose: "An access control policy should be established, documented and reviewed based on business and information security requirements."
    - id: s2.1.1_gdn
      name: guidance
      parts:
      - id: s2.1.1_gdn.1
        name: item
        prose: "Asset owners should determine appropriate access control rules, access rights and restrictions for specific user roles towards their assets, with the amount of detail and the strictness of the controls reflecting the associated information security risks."
      - id: s2.1.1_gdn.2
        name: item
        prose: Access controls are both logical and physical and these should be considered together.
      - id: s2.1.1_gdn.3
        name: item
        prose: Users and service providers should be given a clear statement of the business requirements to be met by access controls.
      - id: s2.1.1_gdn.4
        name: item
        prose: |-
          The policy should take account of the following:

          1. security requirements of business applications;
          2. policies for information dissemination and authorization, e.g. the need-to-know principle and information security levels and classification of information;
          3. consistency between the access rights and information classification policies of systems and networks;
          4. relevant legislation and any contractual obligations regarding limitation of access to data or services;
          5. management of access rights in a distributed and networked environment which recognizes all types of connections available;
          6. segregation of access control roles, e.g. access request, access authorization, access administration;
          7. requirements for formal authorization of access requests;
          8. requirements for periodic review of access rights;
          9. removal of access rights;
          10. archiving of records of all significant events concerning the use and management of user identities and secret authentication information;,
          11. roles with privileged access.
    - id: s2.1.1_stm
      name: information
      parts:
      - id: s2.1.1_stm.1
        name: item
        prose: |-
          Care should be taken when specifying access control rules to consider:

          1. establishing rules based on the premise “Everything is generally forbidden unless expressly permitted” rather than the weaker rule “Everything is generally permitted unless expressly forbidden”;
          2. changes in information labels that are initiated automatically by information processing facilities and those initiated at the discretion of a user;
          3. changes in user permissions that are initiated automatically by the information system and those initiated by an administrator;
          4. rules which require specific approval before enactment and those which do not.
      - id: s2.1.1_stm.2
        name: item
        prose: Access control rules should be supported by formal procedures and defined responsibilities.
      - id: s2.1.1_stm.3
        name: item
        prose: Role based access control is an approach used successfully by many organizations to link access rights with business roles.
      - id: s2.1.1_stm.4
        name: item
        prose: |-
          Two of the frequent principles directing the access control policy are:

          1. Need-to-know: you are only granted access to the information you need to perform your tasks (different tasks/roles mean different need-to-know and hence different access profile);
          2. Need-to-use: you are only granted access to the information processing facilities (IT equipment, applications, procedures, rooms) you need to perform your task/job/role.
```